distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -72,9 +71,17 @@ a {
 
   a {
     color: $normal-text;
-    font-size: 14px;
-    font-weight: bold;
+    font-size: 12px;
     text-decoration: none;
+    line-height: 2.5;
+    display: block;
+    border-radius: 0 4px 4px 0;
+    padding: 0 10px 0 0;
```

Comment:
This is mostly so the text within the links doesn't touch the edge of the gradient. It's `10` but it's also not tied to anything else directly.